### DISCUSSION TITLE
Thank you very much for the free open source program!

### DISCUSSION THREAD

**Discussion by s-r-grass** [Stellarium/stellarium#1340] [General]
Hello, thank you very much for a good program under a free open source license! Added a program to a Russian-language site about free software, here is the [link](https://xn--90abhbolvbbfgb9aje4m.xn--p1ai/%D0%B4%D1%80%D1%83%D0%B3%D0%B8%D0%B5-%D0%BF%D1%80%D0%BE%D0%B3%D1%80%D0%B0%D0%BC%D0%BC%D1%8B/stellarium-%D0%B2%D0%B8%D1%80%D1%82%D1%83%D0%B0%D0%BB%D1%8C%D0%BD%D1%8B%D0%B9-%D0%BF%D0%BB%D0%B0%D0%BD%D0%B5%D1%82%D0%B0%D1%80%D0%B8%D0%B9.html).

The project is interesting, so please don't abandon it!

**Comment by DarthGandalf** [comment]
Names of developers look outdated on that website.
